Utils/Dataflow/pyDKB/dataflow/communication/stream/InputStream.py

```diff
@@ -154,16 +172,22 @@ def get_message(self):
         return result
 
     def next(self):
-        """ Get next message from the input stream.
+        """ Get next message or marker from the input stream.
```

Comment:
> Strings are very.. questionable. Too standard -- so it can be anything.
> 
> _If_ iteration over `InputStream` is to return "various objects from the input", these objects should be wrapped into something unmistakable, so that the external context could know for sure: returned object is not just "some string", it is the `EOBMarker`. It is not necessary objects of some class, it can be constants moved from `communication.*` classes directly to the `communication` module, like `communication.EOBMarker`, `communication.EOMMarker`, `communication.EOPMarker`, etc. Still "strings", but at least they have very precise reference, that will save at least from typo in code.

I agree with this. If we will decide to keep this logic, then changing strings to constants sounds reasonable.

Everything else here is ideological and will be discussed in email/meeting/etc.